eap74-rhel8-payg-multivm/src/main/bicep/mainTemplate.bicep

```diff
@@ -92,6 +88,12 @@ param subnetName string = 'jbosseap-server-subnet'
 @description('Address prefix of the subnet')
 param subnetPrefix string = '10.0.0.0/24'
 
+@description('Name of the existing or new Subnet')
+param subnetForAppGateway string = 'jboss-appgateway-subnet'
```

Comment:
We'll have to revisit this in https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1657652 .